buildpacks/gradle/buildpack.toml

```diff
@@ -12,8 +12,9 @@ clear-env = true
 [[buildpack.licenses]]
 type = "BSD-3-Clause"
 
-[[stacks]]
-id = "*"
```

Comment:
Due to a Pack CLI bug, the wildcard `[[stacks]]` table is still needed (here and in the other `buildpack.toml` files edited in the PR) for the `pack buildpack package` of the CNB release automation to work (will be fixed in the next Pack CLI release), like:
https://github.com/heroku/buildpacks-python/blob/de378063c83e1eec6302e79f93f189162b5bbbbf/buildpack.toml#L14-L17